label_studio/ml_models/urls.py

```diff
@@ -7,8 +7,12 @@
 app_name = 'ml_models'
 router = DefaultRouter()
 router.register(r'models', api.ModelInterfaceAPI, basename='models')
+_api_model_urlpatterns = [
+    path('', views.model_interfaces_view, name='models-list'),
```

Comment:
This isnt an API path, we should be able to go to `{ls_url}/models` to get to this view

Reply:
ok I can remove it then

Reply:
We still need this path registered, just not under `_api_model_urlpatterns` , check our other apps for examples e.g. projects

Reply:
done